app/services/concerns/xccdf/profile_rules.rb

```diff
@@ -7,18 +7,35 @@ module ProfileRules
 
     included do
       def save_profile_rules
```

Comment:
Can you please wrap this in a transaction?

Reply:
Is it really necessary? Isn't the import already running in one?

Reply:
Since this is a public method, and it might be run outside of an import, it might be good idea. DB also support sub-transactions. Anyhow, I leave that to your judgement.